Jenkinsfile

```diff
@@ -218,17 +242,24 @@ pipeline {
                 unstash 'CIScripts'
                 script {
                     try {
-                        unstash 'testReport'
+                        unstash 'testReportsWindowsCompute'
+                        unstash 'testReportsCISelfcheck'
                     } catch (Exception err) {
                         echo "No test report to parse"
                     } finally {
                         powershell script: '''./CIScripts/GenerateTestReport.ps1 `
-                            -XmlsDir test_report `
-                            -OutputDir processed_reports'''
+                            -XmlsDir testReportsRaw/WindowsCompute `
+                            -OutputDir TestReports/WindowsCompute'''
 
-                        dir("processed_reports") {
-                            stash name: 'processedTestReport', allowEmpty: true
-                        }
+                        powershell script: '''./CIScripts/GenerateTestReport.ps1 `
+                            -XmlsDir testReportsRaw/CISelfcheck `
+                            -OutputDir TestReports/CISelfcheck'''
+
+                        // Using robocopy to workaround 260 chars path length limitation.
+                        // TODO: Similar method may be used when CISelfcheck generates detailed logs.
+                        robocopy("${pwd()}/testReportsRaw/WindowsCompute/detailed/", "${pwd()}/TestReports/WindowsCompute/detailedLogs", "*.log")
```

Comment:
The return status of this [`robocopy`] function is ignored.
Also, this command produces lot of output, perhaps we should hide it?